e2e/collections_test/test_contains/test.yml

```diff
@@ -0,0 +1,37 @@
+query: >
+    CALL collections.contains([1,2,3,4,5], 4) YIELD output RETURN output;
+
+output:
+  - output: true
+
+query: >
+    CALL collections.contains([1,2,3,4,5], [3,4])  YIELD output RETURN output;
+
+output:
+  - output: false
```

Comment:
Did you check somehow that this was called? I am not sure if it was by code in "test_module.py" which is calling only one "query". But nevertheless, I would split these in different files as they are effectively different tests, and this was practice until now.